scalafix-tests/src/it/scala/scalafix/tests/URLConfiguration.scala

```diff
@@ -20,4 +20,37 @@ class URLConfiguration
        |
     """.stripMargin
   DiffTest("url", code, "UrlRewrite.stat").foreach(runDiffTest)
+
+  val githubURL =
+    "github:scalacenter/scalafix/1.0"
```

Comment:
Actually, I just realised that we of course just want to unit test this feature. One integration test for the actual http request should be enough. We can create a new tests suite named `GitHubUrlRewriteTest` inside scalafix-tests/src/test/scala/...